somm_proto/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 pub use cosmos_sdk_proto;
-pub mod somm {
+pub mod allocation {
```

Comment:
Note: changing this might require updates in steward.

Reply:
It does, I've made them in my cork PR which depends on this change